dist/filelist.bzl

```diff
@@ -0,0 +1,262 @@
+
+# DO NOT EDIT: automatically built by dist/s_bazel.
+
+# This file is only used by external projects building WiredTiger via Bazel.
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
Package call is reserved for build files which are specific to the directory location, for bazel files which could be loaded anywhere, it should not include Package commands

Reply:
Done.